src/js/components/help/releases-and-artifacts/build-demo-artifact.js

```diff
@@ -1,104 +0,0 @@
-import React from 'react';
-
-import { detectOsIdentifier } from '../../../helpers';
-import CopyCode from '../../common/copy-code';
-
-// we don't support windows yet, so we'll point them to the linux file instead
-const downloadFolder = {
-  Windows: 'linux',
-  MacOs: 'darwin',
-  Unix: 'linux',
-  Linux: 'linux'
-};
-const executable = `
-sudo chmod +x mender-artifact && \\
-sudo mv mender-artifact /usr/local/bin`;
-const artifactGenerator = 'single-file-artifact-gen';
-const artifactName = 'demo-webserver-updated';
-const file_modification = `cat >index.html <<EOF
-Hello World!
-EOF
-`;
-
-export const BuildDemoArtifact = ({ demoArtifactLink, menderVersion = 'master', menderArtifactVersion = 'master' }) => {
```

Comment:
In this particular case there is not a lot of value in keeping this in the UI, for other commands however I could imagine it to be very useful to have a central place to find code snippets that use e.g. the JWT information or the tenant token... But maybe that is becoming more relevant once we have API key support...